app/controllers/games_controller.rb

```diff
@@ -1,2 +1,23 @@
 class GamesController < ApplicationController
+    before_action :set_game
```

Comment:
Make sure you have the right indentation here.

Reply:
before_action :set_Game should also be limited to only the methods needed. e.g. "show", "update", "destroy"